File: Example/PrebidDemoJava/src/main/java/org/prebid/mobile/javademo/utils/ImageUtils.java
package org.prebid.mobile.javademo.utils;

import android.widget.ImageView;

import com.bumptech.glide.Glide;

public class ImageUtils {

    public static void download(String url, ImageView imageView) {
        Glide.with(imageView)
            .load(url)
            .into(imageView);
    }

}

File: Example/PrebidNextGenDemo/src/main/java/org/prebid/mobile/prebidnextgendemo/utils/ImageUtils.kt
package org.prebid.mobile.prebidnextgendemo.utils

import android.widget.ImageView
import com.bumptech.glide.Glide

object ImageUtils {

    fun download(url: String, imageView: ImageView) {
        Glide.with(imageView)
            .load(url)
            .into(imageView)
    }

}